generator/src/main.rs

```diff
@@ -1601,6 +1601,22 @@ impl Parser {
                 }
             }
 
+            impl<'a> From<&'a [sys::ExtensionProperties]> for ExtensionSet {
```

Comment:
Code which isn't dependent on the XML doesn't need to be produced from the generator.